app/stac_api/urls.py

```diff
@@ -13,20 +15,26 @@
 urlpatterns = [
     path(API_BASE, views.landing_page, name='landing-page'),
     path('checker/', views.checker, name='checker'),
-    path(f"{API_BASE}collections/", CollectionList.as_view(), name='collection-list'),
+    path(f"{API_BASE}collections", CollectionList.as_view(), name='collection-list'),
     path(
-        f"{API_BASE}collections/<str:collection_name>/",
+        f"{API_BASE}collections/<collection_name>",
         CollectionDetail.as_view(),
         name='collection-detail'
     ),
+    path(f"{API_BASE}collections/<collection_name>/items", ItemsList.as_view(), name='items-list'),
     path(
-        f"{API_BASE}collections/<str:collection_name>/items",
-        ItemsList.as_view(),
-        name='items-list'
-    ),
-    path(
-        f"{API_BASE}collections/<str:collection_name>/items/<str:item_name>/",
+        f"{API_BASE}collections/<collection_name>/items/<item_name>",
```

Comment:
Is there some kind of validation of the URL parameters? Before there was something like `<str: >`.

Reply:
`<str:>` is the default, so having it defined or not is the same, wihtout it, it makes it a little bit shorter and we don't need to split it over several lines. (See commit message https://github.com/geoadmin/service-stac/pull/57/commits/bb3a2bce74de99eb0e25e5db03b6d02375eb36cd ;-) )